packages/http-server/test/integration/http-server.integration.ts

```diff
@@ -5,13 +5,27 @@
 import {HttpServer} from '../../';
 import {supertest, expect} from '@loopback/testlab';
 import * as makeRequest from 'request-promise-native';
-import {ServerRequest, ServerResponse} from 'http';
+import {ServerRequest, ServerResponse, get} from 'http';
 
 describe('HttpServer (integration)', () => {
   let server: HttpServer | undefined;
 
   afterEach(stopServer);
 
+  it('formats IPv6 url correctly', async () => {
+    server = new HttpServer(dummyRequestHandler);
+    await server.start();
+    if (server.address.family === 'IPv6') {
```

Comment:
IIUC, this test will be effectively skipped if `server.address` returns a different family (typically IPv4), correct?

Can you force the HttpServer instance to listen on IPv6 loopback address `::1` please?

I think the following should work:

```ts
server = new HttpServer(dummyRequestHandler, {host: '::1'});
```

Reply:
Yes, this test is only for IPv6 -> "formats IPv6 url correctly".